src/main/java/net/aufdemrand/denizen/events/entity/EntityKilledScriptEvent.java

```diff
@@ -157,15 +157,17 @@ else if (name.equals("final_damage")) {
         else if (name.equals("cause")) {
             return cause;
         }
-        else if ((name.equals("damager")) && (damager != null)) {
+        else if (name.equals("damager") && damager != null) {
             return damager;
         }
-        else if ((name.equals("projectile")) && (projectile != null)) {
+        else if (name.equals("projectile") && projectile != null) {
             return projectile;
         }
-        for (EntityDamageEvent.DamageModifier dm : EntityDamageEvent.DamageModifier.values()) {
-            if (name.equals("damage_" + dm.name())) {
-                return new Element(event.getDamage(dm));
+        else if (name.startsWith("damage_")) {
+            for (EntityDamageEvent.DamageModifier dm : EntityDamageEvent.DamageModifier.values()) {
+                if (name.equals("damage_" + dm.name())) {

```

Comment:
Assuming you mean the for loop... Otherwise, it is copy-pasta from the old code.